gp-nested-forms/gpnf-triggered-population.php

```diff
@@ -77,8 +77,11 @@ public function output_script() {
 						$form = $( '#gform_{0}'.format( self.formId ) );
 
 						$( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId ) ).find( 'input' ).on( 'change', function() {
-							var value = $( this ).val();
-							if ( value === self.triggerFieldValue || ( value !== '' && self.triggerFieldValue === '_notempty_' ) ) {
+							var input = $( this );
+							var value = input.val();
+							var checked = input[0].checked;
+
+							if ( checked && value === self.triggerFieldValue || ( value !== '' && self.triggerFieldValue === '_notempty_' ) ) {
```

Comment:
Does this `checked && value === self.triggerFieldValue` need parentheses?

Reply:
Ooh, yeah probably

Reply:
Added in because I _think_ it makes it a bit more robust and still works correctly with the added parens